src/app/containers/Ad/index.jsx

```diff
@@ -16,9 +18,15 @@ const AdContainer = () => {
   }
 
   if (isAmp) {
-    return <Amp />;
+    return <AmpAd />;
   }
-  return null;
+
+  return (
+    <>
+      <CanonicalAdBootstrap />
```

Comment:
Howcome the the ad bootstrap isn't just in the CanonicalAd component?

Reply:
I think either way we need to put some consideration into moving this out of the component completely, if there are multiple ads we don't want it running multiple times.

I think this makes it a little clearer that it isn't supposed to be embedded in the Ad. If that makes any sense?